### ISSUE TITLE
White Border/Background on View is transparent

### ISSUE THREAD

**Issue by EnglederLucas** [expo/expo#11261]
### Summary of Issue 
When I try to create a View with a white border or background the resulting View is not white but transparent. This bug only occurs when I run the Expo app on my Android device. In Expo Web it looks perfectly fine.
The border is still visible but its not the desired color. Strangely, I've tried some other colors (green, blue,red, orange) and they all worked, but white and yellow were either transparent or only barely visible.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

```
  Expo CLI 4.0.8 environment info:
    System:
      OS: Windows 10 10.0.18363
    Binaries:
      Node: 14.15.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 7.0.10 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  4.1.0.0 AI-201.8743.12.41.6858069
    npmPackages:
      expo: ^39.0.3 => 39.0.3
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz => 0.63.2
      react-native-web: ~0.13.12 => 0.13.16
    Expo Workflow: managed
```

Android

### Reproducible Demo

Basically only this: 

```JSX
<View
      style={
        {
          backgroundColor: "white",
          borderWidth: 2,
        },
      }></View>
```

 
### Steps to Reproduce

1. Create a View
2. Add style with borderColor: "white" (or "#fff")
3. Open the Expo App on Android

### Expected Behavior vs Actual Behavior
The borderColor/backgroundColor should be white, not transparent.

![expo](https://user-images.githubusercontent.com/26169816/101419288-fbeec300-38ef-11eb-806d-2d8af2a34efb.png)

It works with other colors
<img src="https://user-images.githubusercontent.com/26169816/101419295-00b37700-38f0-11eb-8bf3-a5d77fb9c1cb.jpeg" width="200" height="400">

**Comment by byCedric** [comment]
Hi @EnglederLucas, thanks for the issue report. Unfortunately, I'm not able to reproduce this issue using your example. You can [find my snack here](https://snack.expo.io/@bycedric/expo-issue-11261), which combines the default code and your example into one. Both the `borderColor` + `backgroundColor`, and the `borderColor` without `backgroundColor` yields the expected results on my Android (10) phone.

If you think this is an issue with Expo, please provide a small repro where the issue occurs. Only with that, we are able to investigate further.

Hope this helps!

**Comment by EnglederLucas** [comment]
O., Thanks for the help. But it seems to have something to do with my phone. I just tried your Snack with my Xiaomi MI 9T with MIUI 12 and I had the same problem.

<img src="https://user-images.githubusercontent.com/26169816/101523927-8e8f7080-3989-11eb-8622-5d9fae134305.jpeg" width="30%" height="30%">

**Comment by byCedric** [comment]
Hmm, that's interesting! I can slightly see the background color but it looks like it's transparent somehow. Did you edit some of the snack code I sent in your screenshot?

**Comment by EnglederLucas** [comment]
No, absolutely not. I just scanned the QR Code on your Snack and ran it on my device.

**Comment by EnglederLucas** [comment]
Interesting, I just remembered that MIUI implements a **dark mode**. I deactivated it and there we go. But I still would like to force the style. Even with activated dark mode.

<div style="display: flex; flex-direction: 'row'">
<img src="https://user-images.githubusercontent.com/26169816/101533460-24c99380-3996-11eb-9e4f-47e59afd7e8f.jpeg"  width="20%" height="20%"/>
<img src="https://user-images.githubusercontent.com/26169816/101533465-25fac080-3996-11eb-86e7-6ad692e6add3.jpeg"  width="20%" height="20%"/>
<div>

**Comment by byCedric** [comment]
Ah, it's probably that MIUI feature that's messing with the styles. One thing you could try is ejecting to bare (you can do that in a separate branch and don't have to commit it), and add `<item name="android:forceDarkAllowed">false</item>` to the Android config. That might help validating if this is the issue you encounter. (see [this SO thread](https://stackoverflow.com/questions/62696305/theme-problem-when-dark-mode-is-activated-under-miui-11-12))

Once validated, we can check if and what we can do to help!

**Comment by joaolucasfarias** [comment]
That's exactly the problem, @byCedric. In another repo, you have a brief discussion and a link to a solution: https://github.com/expo/react-native-appearance/issues/60

I'm facing the same problem right now. Can you guys do something to help us?

**Comment by joaolucasfarias** [comment]
That's exactly the problem, @byCedric. In another repo, you have a brief discussion in [this opened issue](https://github.com/expo/react-native-appearance/issues/60) and a link to a solution

I'm facing the same problem right now. Can you guys do something to help us?

**Comment by WillLopes16** [comment]
I'm facing the same problem with my device I lose, is there a solution already available without having to eject EXPO? Ejecting is not an option for me! any solution prediction?

**Comment by rsallar** [comment]
Hey guys, facing the same problem. Any solution so far without ejecting? 
I've tried SystemUI.setBackgroundColorAsync(color) but didn't work.

**Comment by Hirbod** [comment]
I am also facing this issue right now. I would kindly ask for something like `android.forceDarkMode: false` as a app-config setting. Or maybe an expo-config-plugin? Not sure if the API has something ready for this.

**Comment by Hirbod** [comment]
In case anybody needs it: I've created a expo-config-plugin to fix this issue.
https://gist.github.com/Hirbod/d6bb5d0fc946a12ba9e3cf01120c604a

**Comment by SrBrahma** [comment]
@byCedric can you guys add @hirbod solution to app.json? That was a little hard to find. This automatic dark mode is never good.

**Comment by hirbod** [comment]
Not stale.

**Comment by hirbod** [comment]
Still not stale

**Comment by Abel13** [comment]
> In case anybody needs it: I've created a expo-config-plugin to fix this issue for custom-dev-client users. https://gist.github.com/Hirbod/d6bb5d0fc946a12ba9e3cf01120c604a

Thank you! it worked for me... I just changed:
`value: "false",`
to
`value: false`,

**Comment by vyshnav-chikku** [comment]
disable dark mode on your phone

**Comment by AnatuGreen** [comment]
> disable dark mode on your phone

No. That is not a good solution. I am not building an app that only I will use. Am I supposed to tell everyone in the world who uses MI device to disable dark mode on their phone to get the best of my UI?
There has to be a solution to this somehow. We are devs. That is what we do - find solutions.

**Comment by AnatuGreen** [comment]
I realized that after applying the solutions above, one needs to rebuild the app for the light mode to be applied. The dark mode persists in Expo Go but when you build your expo dev client and install it you will get the desired result
